sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -332,6 +332,11 @@ def test_upretty_modifiers():
     assert upretty( Symbol('x__dot') ) == u('x__dot')
 
 
+def test_pretty_Cycle():
+    from sympy.combinatorics.permutations import Cycle
+    assert pretty(Cycle(1, 2)) == '([1, 2],)'
```

Comment:
Sorry, if i understand it properly then.
Since, `str()` is supposed to print it as it is(already mentioned). Then i guess the printer(non-pretty) supposed to print is as `(1 2)` has to be `srepr`?
```
>>> srepr(Cycle(1, 2, 4)(5, 6)
'(1 2 4)(5 6)'   # you expect this way?
```
(Consider this only if above statement is right) If that is the case, then is it not quite a lot different from usually how `srepr` works like for `x + y` it prints `Add(Symbol('x', Symbol('y'))`?